optaweb-employee-rostering-benchmark/pom.xml

```diff
@@ -103,11 +107,23 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-test</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.junit.vintage</groupId>
+          <artifactId>junit-vintage-engine</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
Please remove the `spring-boot-test` and `spring-test` dependencies above. They're already provided by `spring-boot-starter-test`. Doing that will make it unnecessary to exclude the vintage engine twice.